components/calendar/appearance.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Appearance
+page_title: Calendar Appearance
+description: Appearance settings of the Calendar for Blazor.
+slug: calendar-appearance
+tags: telerik,blazor,calendar,appearance
+published: True
+position: 30
+---
+
+# Appearance Settings
+
+You can control the appearance of the Calendar by setting the following attribute:
+
+## Size
+
+You can increase or decrease the size of the Calendar by setting the `Size` attribute to a member of the `Telerik.Blazor.ThemeConstants.Calendar.Size` class:
+
+| Class members | Manual declarations |
+|------------|--------|
```

Comment:
We are not aiming for ASCII-art style, so better use the minimum number of hyphens.

```suggestion
|---|---|
```